MetaMask/metamask-mobile
# fix: Add missing long press event for account removal

**Description**

This PR re-adds the ability to remove imported accounts via long press

**Screenshots/Recordings**

Steps to verify the fix:
- Import an account
- Open account selection on either the wallet or browser screen
- Long press the imported account and you should see the remove prompt
- Proceed to remove account
- Imported account should be successfully removed

Removing from wallet screen
https://github.com/MetaMask/metamask-mobile/assets/10508597/b873556a-6649-4733-8013-99d7b2124501

Removing from browser screen
https://github.com/MetaMask/metamask-mobile/assets/10508597/bfebbe93-57b5-4f26-9ecf-2859ad1a59ef

**Issue**

#6985

**Checklist**
